cob_base_controller_utils/src/stop_detector.cpp

```diff
@@ -100,11 +129,14 @@ int main(int argc, char* argv[])
     g_recovered = true;
     g_last_received = ros::Time::now();
     g_timeout = ros::Duration(timeout);
+    g_controller_spawn = {"joint_state_controller", "twist_controller", "odometry_controller"};
```

Comment:
the `odometry_controller` must NOT be restarted - otherwise localization might get disrupted
this is modified locally on mrk-5